app/javascript/tests/layout/Header.test.js

```diff
@@ -0,0 +1,173 @@
+    expect(mainNav.find({ children: 'Enroller Dashboard' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'Monitoring Dashboards' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'Admin Panel' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'Analytics' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'API' }).exists()).toBe(false);
+    expect(mainNav.find({ children: 'Jobs' }).exists()).toBe(false);
+    expect(wrapper.find(Navbar).find({ bg: 'warning' }).exists()).toBe(false);
+  });
+
+  it('Properly renders Enroller Dashboard tab', () => {
+    mockUser1.can_see_enroller_dashboard_tab = true;
+    const wrapper = getWrapper({ current_user: mockUser1 });
+    const mainNav = getMainNav(wrapper);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/patients` })
+        .exists()
+    ).toBe(true);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/patients` })
+        .text()
+    ).toEqual('Enroller Dashboard');
+  });
+
+  it('Properly renders Analytics tab', () => {
+    mockUser1.can_see_analytics_tab = true;
+    const wrapper = getWrapper({ current_user: mockUser1 });
+    const mainNav = getMainNav(wrapper);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/analytics` })
+        .exists()
+    ).toBe(true);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/analytics` })
+        .text()
+    ).toEqual('Analytics');
+  });
+
+  it('Properly renders Monitoring Dashboards tab', () => {
+    mockUser1.can_see_monitoring_dashboards_tab = true;
+    const wrapper = getWrapper({ current_user: mockUser1 });
+    const mainNav = getMainNav(wrapper);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/public_health` })
+        .exists()
+    ).toBe(true);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/public_health` })
+        .text()
+    ).toEqual('Monitoring Dashboards');
+  });
+
+  it('Properly renders USA admin tabs', () => {
+    mockUser1.is_usa_admin = true;
+    const wrapper = getWrapper({ current_user: mockUser1 });
+    const mainNav = getMainNav(wrapper);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/oauth/applications` })
+        .exists()
+    ).toBe(true);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/oauth/applications` })
+        .text()
+    ).toEqual('API');
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/sidekiq` })
+        .exists()
+    ).toBe(true);
+    expect(
+      mainNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/sidekiq` })
+        .text()
+    ).toEqual('Jobs');
+  });
+
```

Comment:
I'd also add a test that that tests the activeKey

Reply:
[a72d21f](https://github.com/SaraAlert/SaraAlert/pull/1243/commits/a72d21f19ad06cdee88b9ee3cbb159a387cc453c)